src/Curriculum/Survey/Survey.tsx

```diff
@@ -147,6 +151,13 @@ const Survey = () => {
         setSurveyAnswers(surveyAnswers);
     };
 
+    const submitSurvey = () => {
+        if (data?.items.length !== answeredIndices.size) {
+          alert('You answered ' + answeredIndices.size + ' out of ' + data?.items.length + ' questions!');
```

Comment:
You do not have to change it but this can also be written

```suggestion
          alert(`You answered ${answeredIndices.size}  out of  ${data?.items.length} questions!`);
```